backend/src/bundles/profile/profile.repository.ts

```diff
@@ -12,6 +12,13 @@ class ProfileRepository extends AbstractRepository<
     }: Record<'profileModel', typeof ProfileModel>) {
         super(profileModel);
     }
+
+    public async updateById(
+        id: string,
+        data: Partial<Omit<ProfileModel, 'id' | 'createdAt' | 'updatedAt'>>,
+    ): Promise<ProfileModel> {
+        return await this.model.query().updateAndFetchById(id, data);
+    }
```

Comment:
since we are extending the AbstractRepo, find a way to reuse this method for both userRepo and profileRepo